### PR TITLE
expression: implement vectorized evaluation for `builtinLog10Sig`

### PR DESCRIPTION
**Pull request by qw4990**
### What problem does this PR solve? 
Implement vectorized evaluation for `builtinLog10Sig`

### What is changed and how it works?
Here is the benchmark, almost 2.5x fater than before:
```
BenchmarkVectorizedBuiltinFunc/builtinLog10Sig-VecBuiltinFunc-12                          200000              8148 ns/op
BenchmarkVectorizedBuiltinFunc/builtinLog10Sig-NonVecBuiltinFunc-12                       100000             20654 ns/op
```

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: expression/builtin_math_vec.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
+// Copyright 2013 The ql Authors. All rights reserved.
```
Comment (zz-jason) [inline expression/builtin_math_vec.go]:
```suggestion
// Copyright 2019 The ql Authors. All rights reserved.
```

File: expression/builtin_math_vec.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	"github.com/pingcap/tidb/util/chunk"
+)
+
+func (b *builtinLog10Sig) vecEvalReal(input *chunk.Chunk, result *chunk.Column) error {
+	if err := b.args[0].VecEvalReal(b.ctx, input, result); err != nil {
+		return err
+	}
+	f64s := result.Float64s()
+	for i := 0; i < len(f64s); i++ {
+		if result.IsNull(i) {
```
Comment (zz-jason) [inline expression/builtin_math_vec.go]: would it be faster if we don't check `null` here?

File: expression/builtin_math_vec.go (hunk 1 above)
Comment (SunRunAway) [reply expression/builtin_math_vec.go]: Unnecessary to state `ql` license here?

Comment (qw4990) [reply expression/builtin_math_vec.go]: Yes, it's my mistake.
It's updated now.

File: expression/builtin_math_vec.go (hunk 2 above)
Comment (qw4990) [reply expression/builtin_math_vec.go]: After testing, this way is slightly faster.
I think the reason is that `math.Log10` is relatively heavy to `SetNull`, so it's better to check nulls here.

**Comment by zz-jason** [comment]
to https://github.com/pingcap/tidb/issues/12058

**Pull request merged**